jekyll/_cci2/migrating-from-launch-agent-to-the-machine-runner-agent-3-0-preview.adoc

```diff
@@ -0,0 +1,89 @@
+
+If the agent has been installed as a `systemd` service, run the following commands to stop and uninstall the service unit, and delete the service file (the new agent will install a new service unit):
++
+```shell
+sudo systemctl stop circleci.service
+sudo systemctl disable circleci.service
+
+sudo rm /usr/lib/systemd/system/circleci.service
+```
+
+. The launch agent binary can now be removed from the machine. The default install location is `/opt/circleci/circleci-launch-agent`, but this may differ on your machine if the agent was installed somewhere else. The entire `/opt/circleci` subdirectory can be removed at this point.
+
+. Finally, move the agent configuration file. The default location for this is `/etc/opt/circleci/launch-agent-config.yaml`. Move the file to `/etc/circleci-runner/circleci-runner-config.yaml` using the following command:
++
+```shell
+mv /etc/opt/circleci/launch-agent-config.yaml /etc/circleci-runner/circleci-runner-config.yaml
+```
++
+Alternatively, you can let the machine agent install process create a config file template and copy the existing config into the newly created file (CROSSLINK)
+
+[#uninstalling-launch-agent-linux-se]
+==== RHEL/Rocky Linux
+
+In addition to the steps listed above, the SELinux policy should be uninstalled using the following command:
+
+```shell
+sudo semodule -r circleci_launch_agent
+```
+
+[#install-machine-agent]
+== 2. Installing the Machine Agent
+
+Packaged versions of the machine runner 3.0 preview are available for Linux for a streamlined installation experience.
+
+. At present deb and rpm packages are available for linux. Use the following commands to install them
++
+```shell
+# DEB Commands
+# This script installs the package repository to the package manager
+curl -s https://packagecloud.io/install/repositories/circleci/runner/script.deb.sh | sudo bash
+sudo apt install circleci-runner
+```
++
+```shell
+# RPM Commands
+# This script installs the package repository to the package manager
+curl -s https://packagecloud.io/install/repositories/circleci/runner/script.rpm.sh | sudo bash
+sudo yum install circleci-runner
+```
++
+The packages installs the latest version of the agent. They also create a circleci user as well as required directories for operation of the agent.
+
+. The packages creates a template configuration file at `/etc/circleci-runner/circleci-runner-config.yaml`. This configuration is 1:1 compatible with the launch agent configuration, apart from the auto-update feature. Since auto-update has been removed from the new agent, you should use the appropriate package management update tools to get new versions of the package when they are released. When using the `apt` package to update the agent package, you will need to clear the configuration file modified prompt. This can be avoided using the `--force-confold` flag or by removing the modified configuration file before upgrading. 
```

Comment:
```suggestion
. The packages create a template configuration file at `/etc/circleci-runner/circleci-runner-config.yaml`. This configuration is 1:1 compatible with the launch agent configuration, apart from the auto-update feature. Since auto-update has been removed from the new agent, you should use the appropriate package management update tools to get new versions of the package when they are released. When using the `apt` package to update the agent package, you will need to clear the configuration file modified prompt. This can be avoided using the `--force-confold` flag or by removing the modified configuration file before upgrading. 
```